### ISSUE TITLE
8416 Staging - Review screen > Focus management needed for list and loop

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-team#122501]
## Description
On the review screen, focus management for the list and loop should match the focus management inside the form flow:

**Edit**
When editing a list and loop item, after the user is done, focus needs to be brought to the success message on the review screen. This is happening very briefly but then focus is lost for some reason. This doesn't happen in the standard edit flow, just when editing from the review page.

**Cancel**
When editing a list and loop item and then deciding to cancel, after the user is done, focus should be brought back to the h4 "Review" heading. Currently focus goes to the H2 stepper

**Delete**
When Deleting a list and loop item, focus should be to be brought to the alert so users know their action was completed successfully. Currently focus is going to the "Add another" button

### Acceptance criteria

- [ ] Focus management for editing a list and loop item from the Review screen matches focus management outside the review screen
- [ ] Focus management for canceling a list and loop item from the Review screen matches focus management outside the review screen
- [ ] Focus management for deleting a list and loop item from the Review screen matches focus management outside the review screen
